lib/metrics/services.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
 	'barriers-api-direct': /^https?:\/\/barrier-app(-glb)?\.memb\.ft\.com\/memb\/barrier/,
 	'barriers-guru-individual-api': /^https?:\/\/barrier-guru\.ft\.com\/individual/,
 	'barriers-guru-corporate-api': /^https?:\/\/barrier-guru\.ft\.com\/corporate/,
-	'barrier-guru-api': /^https?:\/\/barrier-guru\.ft\.com\/barrier/,
+	'barrier-guru-api': /^https?:\/\/barrier-guru\.ft\.com\/preflight/,
```

Comment:
We need both /preflight/ (called from preflight) and /barrier/ (called from next-product)
```suggestion
	'barrier-guru-api': /^https?:\/\/barrier-guru\.ft\.com\/barrier/,
	'barrier-guru-api-preflight': /^https?:\/\/barrier-guru\.ft\.com\/preflight/,
```

Reply:
@ludrob 
I agree we need both but I was not able to find where the metrics are defined for next product.?

If we add the above line then we need to update next-preflight as well to update the service metric name as well.

Reply:
I would suggest another way around:

```
'barrier-guru-api': /^https?:\/\/barrier-guru\.ft\.com\/preflight/,
'barrier-guru-api-product': /^https?:\/\/barrier-guru\.ft\.com\/barrier/,
```

Reply:
Good point, next-preflight uses the metric key here: https://github.com/Financial-Times/next-preflight/blob/ad2ebad0485ee28f620245ca311dc48746c195f9/server/config/downstream-perf.js#L59

Let's flip it around then
```suggestion
	'barrier-guru-api': /^https?:\/\/barrier-guru\.ft\.com\/preflight/,
	'barrier-guru-api-product': /^https?:\/\/barrier-guru\.ft\.com\/barrier/,
```

As far as I know we do not directly reference the metric key directly anywhere in next-product.
So we just need to have the url registered here and should be fine, the key should not matter.